examples/uefi-demo/main.rs

```diff
@@ -229,6 +352,52 @@ impl slint::platform::Platform for Platform {
                 });
                 self.window.dispatch_event(slint::platform::WindowEvent::KeyReleased { text });
             }
+            // mouse handle until no input
+            while let Some(mut mouse) =
+                mpointer.read_state().expect("Failed to read state from Pointer.")
+            {
+                position.x += (mouse.relative_movement.0 as f32) / (mouse_mode.resolution.0 as f32);
+                position.y += (mouse.relative_movement.1 as f32) / (mouse_mode.resolution.1 as f32);
+
+                let button: slint::platform::PointerEventButton = match mouse.button {
+                    (true, true) => slint::platform::PointerEventButton::Left,
+                    (true, false) => slint::platform::PointerEventButton::Left,
+                    (false, true) => slint::platform::PointerEventButton::Right,
+                    (false, false) => slint::platform::PointerEventButton::Other,
+                };
+
+                if position.x < 0.0 {
+                    position.x = 0.0;
+                } else if position.x > (info.resolution().0 - pointer_x) as f32 {
+                    position.x = (info.resolution().0 - pointer_x - BOUND_SPACE) as f32;
+                    mouse.relative_movement.0 = (info.resolution().0 - BOUND_SPACE) as i32;
+                }
+
+                if position.y < 0.0 {
+                    position.y = 0.0;
+                } else if position.y > (info.resolution().1 - pointer_y) as f32 {
+                    position.y = (info.resolution().1 - pointer_y - BOUND_SPACE) as f32;
+                    mouse.relative_movement.1 = (info.resolution().1 - BOUND_SPACE) as i32;
+                }
+
+                self.window.dispatch_event(slint::platform::WindowEvent::PointerMoved { position });
+                self.window.dispatch_event(slint::platform::WindowEvent::PointerExited {});
+
+                self.window.dispatch_event(slint::platform::WindowEvent::PointerPressed {
+                    position,
+                    button,
+                });
+                self.window.dispatch_event(slint::platform::WindowEvent::PointerReleased {
+                    position,
+                    button,
+                });
+                is_mouse_move = true;
+            }
+
+            if is_mouse_move {
+                self.window.request_redraw();
```

Comment:
this doesn't mark the region dirty. I think that is missing.

Reply:
when mouse pointer is moving, the slint-ui doesn't feel it. so I must request redraw. or the mouse would move everywhere makes screen dirty while slint-ui no refresh it.
So I suggest that slint-ui should provide the mouse pointer. The better way which may as [tronical](https://github.com/tronical)  say https://github.com/slint-ui/slint/pull/5034#issuecomment-2054159637